lib/chef/provisioning/docker_driver/docker_transport.rb

```diff
@@ -179,11 +179,12 @@ def upload_file(local_path, path)
     def make_url_available_to_remote(url)
       # The host is already open to the container.  Just find out its address and return it!
       uri = URI(url)
+      uri.scheme = 'http' if 'chefzero' == uri.scheme && uri.host == 'localhost'
```

Comment:
Didn't work for me with chef-provisioning 1.2 and this line removed. `url` always comes as `chefzero://localhost:8889` if chef zero used as provisioner. Just in case `gem list`:
```
chef (12.3.0)
chef-provisioning (1.2.0)
chef-zero (4.2.1)
cheffish (1.2)
```